apps/web/src/components/NavMenu/MainNavMenu.tsx

```diff
@@ -157,6 +157,13 @@ const MainNavMenu = () => {
                     color={isSmallScreen ? "black" : "whiteFixed"}
                     mode="text"
                     block={false}
+                    aria-label={intl.formatMessage({
+                      defaultMessage:
+                        "Select a different role to access its corresponding tools and information",
+                      id: "sSbt7W",
+                      description:
+                        "Aria text to describe what the purpose of the role menu is",
+                    })}
```

Comment:
So, it looks like `aria-description` is not ready for production yet :cry: So not using it was the right call. However, with `aria-label`, we no longer know what the current role is. 

We should get both contexts for this and right now, I am not sure how we can do that :thinking:

Reply:
Tricky, tricky then huh